src/openforms/variables/base.py

```diff
@@ -1,7 +1,9 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
+from typing import Optional
 
 from openforms.forms.models import FormVariable
+from openforms.submissions.models import Submission
```

Comment:
```suggestion
if TYPE_CHECKING:
    from openforms.submissions.models import Submission
```
Since it's only used in the type hints?
(=> then quotes around "Submission" in the type hints)

Reply:
the `TYPE_CHECKING` workaround is mostly used to solve circular imports, if it works without, that leads to cleaner code.